__tests__/app.test.js

```diff
@@ -0,0 +1,51 @@
+const app = require("../app");
+const request = require("supertest");
+
+const {
+  articleData,
+  commentData,
+  topicData,
+  userData,
+} = require("../db/data/test-data/index");
+const db = require("../db/connection");
+const seed = require("../db/seeds/seed");
+
+beforeEach(() => {
+  return seed({ articleData, commentData, topicData, userData }).then(() => {});
+});
+
+afterAll(() => {
+  return db.end();
+});
+
+describe("task 2. GET /api/topics", () => {
+  test("GET 200 /api/topics. Has to return an arr of objs. Each obj has to have description and slug properties", () => {
+    return request(app)
+      .get("/api/topics")
+      .expect(200)
+      .then(({ body }) => {
```

Comment:
If you wanted to destructure further, you could absolutely use ```{body:{topics}}``` and then access the topics directly.